nautobot/dcim/tests/test_filters.py

```diff
@@ -1234,10 +1230,10 @@ def test_u_height(self):
 
     def test_desc_units(self):
         with self.subTest():
-            params = {"desc_units": "true"}
+            params = {"desc_units": True}
             self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
         with self.subTest():
-            params = {"desc_units": "false"}
+            params = {"desc_units": False}
```

Comment:
Why this change? I see you've made it in a bunch of other cases below as well.

Reply:
Consistency.